chapters/en/chapter10/7.mdx

```diff
@@ -0,0 +1,183 @@
+<!-- DISABLE-FRONTMATTER-SECTIONS -->
+
+# End-of-chapter quiz[[end-of-chapter-quiz]]
+
+<CourseFloatingBanner
+    chapter={9}
+    classNames="absolute z-10 right-0 top-0"
+/>
+
+Let's test what you learned in this chapter!
+
+### 1. What can you use Argilla for?
+
+<Question
+	choices={[
```

Comment:
You could add "Generate synthetic datasets" and then mention distilabel, people always mess up with that